radio/src/gui/colorlcd/model_select.cpp

```diff
@@ -472,6 +494,7 @@ void ModelsPageBody::editLabels(ModelCell* model)
 void ModelsPageBody::update(int selected)
 {
   clear();
+  currentModelButton = nullptr;  // will soon have new buttons
```

Comment:
you can moce this to where curModel is created, because the loop is the first use of that variable.

Reply:
currentModelButton is a class level variable. Since the ModelButton's get recreated each time update is called, currentModelButton has to be reset each time update is called and the ModelButton's are re-created.

Reply:
ok